src/lib/prisma-plugins/n1-detector.ts

```
@@ -0,0 +1,144 @@
+  );
+
+  if (!relevantLine) {
+    return undefined;
+  }
+
+  // This regex is an attempt and might need refinement
+  const match = relevantLine.match(/at\s+(?:async\s)?(?:([\w\.$<>\[\]\s]+)\s\()?(.+?):(\d+):(\d+)\)?/);
+
+  if (match) {
+    const filePath = match[2] || match[1]; // Path can be in group 1 or 2
+    const lineNumber = parseInt(match[3], 10);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)